content/en/blog/_posts/2016-08-00-Security-Best-Practices-Kubernetes-Deployment.md

```diff
@@ -205,7 +205,15 @@ Reference [here](http://kubernetes.io/docs/api-reference/v1/definitions/#_v1_pod
 
 
 
```

Comment:
wasn't sure how revisionist we get about blog post updates, but this advice isn't current

Reply:
The ResourceQuota section is incorrect, also

Reply:
Maybe just mark the blog post as being out of date and containing errors?